pyeudiw/tests/satosa/test_backend.py

```diff
@@ -569,12 +571,19 @@ def test_request_endpoint(self, context):
             HTTP_DPOP=dpop_proof
         )
 
+        state = self.backend.state_endpoint(context)
+        assert state.status == "202"
+        assert state.message
+        msg = json.loads(state.message)
+        assert msg["response"] == "Not completed"
+
+        context.request_method = "POST"
+        context.request_uri = CONFIG['metadata']['request_uris'][0] + f"/{context.state['SESSION_ID']}"
```

Comment:
here we should use a urlparameter, don't append another webpath otherwise the regexp to bound the endpoint should be changed

the urlparameter then will be taken from the context